packages/remix/migrations.json

```diff
@@ -120,6 +120,31 @@
         "version": "^1.15.0",
         "alwaysAddToPackageJson": true
       }
+    },
+    "16.3.0": {
+      "version": "16.3.0",
```

Comment:
@philipjfulcher @jaysoo I'm not sure what version we should target for this, but I've selected the next minor for now.